dist/sendMoney.js

```diff
@@ -35,6 +35,10 @@ const errors = {
   INVALID_ACCOUNT_NUMBER: {
     code: 'sendMoney-8',
     message: 'Invalid account number'
+  },
+  TRANSACTION_TO_INTERNAL_ACCOUNT_HAS_TIMED_OUT: {
```

Comment:
que le parece `INTERNAL_ACCOUNT_TRANSACTION_TIMEOUT`? 
y en description `Internal transaction failed due to connection timeout error.`?
mas cortos y simplificados, pero son meras sugerencias si ve que estan feos podemos dejar estos 👍